src/components/ServerList/ServerList.scss

```diff
@@ -0,0 +1,98 @@
+.server-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  gap: 10px;
+  min-height: 100vh;
+  width: 100px;
+  padding: 20px;
+  background-color: var(--grey-color-darker);
+
+  &--discord {
+    img {
+      width: 100%;
+    }
+  }
+
+  &--line {
+    width: 40px;
+    height: 2px;
+    background-color: var(--grey-color-dark-02);
+  }
+
+  &--user {
+    position: relative;
+
+    img {
+      width: 100%;
+      height: 60px;
+      border-radius: 50%;
+
+      &:hover {
+        border-radius: 30%;
+        transition: border-radius 0.2s ease-in;
+      }
+    }
+
+    &::after {
+      position: absolute;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      min-width: 20px;
+      height: 20px;
+      content: "2";
+      bottom: 0;
+      right: 0;
+      padding: 1px;
+      font-weight: 700;
+      border-radius: 20px;
+      border: 3px solid black;
+      background-color: red;
+    }
+  }
+
+  &--add-server {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 60px;
+    min-height: 60px;
+    border-radius: 50%;
+    background-color: var(--grey-color-dark);
+
+    &:hover {
+      border-radius: 30%;
+      transition: border-radius 0.2s ease-in;
+      background-color: var(--green-color-dark-01);
+    }
+  }
+
+  &--download-data {
+    display: flex;
```

Comment:
```
{
    display: flex;
    align-items: center;
    justify-content: center;
    width: 60px;
    min-height: 60px;
    border-radius: 50%;
    background-color: var(--grey-color-dark);

    &:hover {
      border-radius: 30%;
      transition: border-radius 0.2s ease-in;
      background-color: var(--green-color-dark-01);
    }
  }
```

tot codul asta il vad aplicat si la **server-list--add-server**

as redenumi clasa in 

.item (stii ca esti in contextul de **.server-list**)

iar unde ai 

 <div className="server-list--download-data">
        <img src={plus} fill="#000000" />
      </div>

pui 2 clase, item si download-data
```
 <div className="item download-data">
        <img src={plus} fill="#000000" />
      </div>
```